build/gradle/commitAndProductionStage.gradle

```diff
@@ -676,156 +660,6 @@ def packageClient(productOutputPath) {
-
-        // copy service launcher
-        def serviceLauncher = project.ext.serviceExeName + '.exe'
-        def serviceWrapper = (Helper.buildFor64bit(project) ? 'jsl64.exe' : 'jsl.exe')
-        project.copy {
-            from new File(srcPath, serviceWrapper)
-            into productOutputPath
-            rename { serviceLauncher }
-        }
-
-        // read command line from ini file
-        def iniFile = new File(productOutputPath, "${launcherName}.ini").text
-        def programArgs = iniFile.replaceAll(/\r?\n+/, ' ').replaceAll(/-vmargs.*/, '').trim()
-        def vmArgs = iniFile.replaceAll(/\r?\n+/, ' ').replaceAll(/.*-vmargs/, '').trim()
-
-        // find launcher jar
-        def launcherJar = 'plugins/' + fileTree(dir: new File(productOutputPath, 'plugins'),
-                include: 'org.eclipse.equinox.launcher*.jar').files.toList()[0].name
-
-        def values = [
-            'launcher': launcherName,
-            'serviceName': project.ext.serviceName,
-            'serviceShort': project.ext.serviceExeName,
-            'cmdLine': "${vmArgs} -jar ${launcherJar} ${programArgs}"
-        ]
-        renderTemplate(new File(srcPath, 'service-install.bat'), new File(productOutputPath, 'service-install.bat'), values)
-        renderTemplate(new File(srcPath, 'service-remove.bat'), new File(productOutputPath, 'service-remove.bat'), values)
-        renderTemplate(new File(srcPath, 'jsl.ini'), new File(productOutputPath, "${project.ext.serviceExeName}.ini"), values)
-    }
-
-    // create package
-    def packageName = getPackageName()
-    def packageFileName
-    if (Helper.buildForWindows(project)) {
-        packageFileName = new File("target/${packageName}.zip")
-    } else {
-        packageFileName = new File("target/${packageName}.tar.gz")
-    }
-
-    println("Creating package ${packageFileName} ...")
-    if (Helper.buildForWindows(project)) {
-        ant.zip(destfile: packageFileName) {
-            zipfileset(dir: productOutputPath, prefix: packageName)
-        }
-    } else {
-        ant.tar(destfile: packageFileName, longfile: 'gnu', compression: 'gzip') {
-            tarfileset(dir: productOutputPath, prefix: packageName) {
-                exclude(name: launcherName)
-                exclude(name: '**/*.sh')
-                exclude(name: '**/*.so')
-                exclude(name: '**/jre/bin/*')
-				exclude(name: launcherName)
-				exclude(name: "${productName}.app/Contents/MacOS/${launcherName}")
-            }
-            tarfileset(dir: productOutputPath, prefix: packageName, filemode: '755') {
-                include(name: launcherName)
-                include(name: '**/*.sh')
-                include(name: '**/*.so')
-                include(name: '**/jre/bin/*')
-				include(name: launcherName)
-				include(name: "${productName}.app/Contents/MacOS/${launcherName}")
-            }
-        }
-
-        // Docker image
-        // right now mainly intended for Infocenter product
```

Comment:
Isn't the Infocenter product also a server build? Does it still work after removing all the build logic?
If it's not needed for the Infocenter, then it can be removed from my point of view.

Reply:
Thanks, I restored it. It is indeed needed for the Infocenter build.